haskell-src/Concordium/Types/Execution.hs

```diff
@@ -356,7 +356,9 @@ data Payload
           -- | The commission the pool owner takes on baking rewards.
           cbBakingRewardCommission :: !(Maybe AmountFraction),
           -- | The commission the pool owner takes on finalization rewards.
-          cbFinalizationRewardCommission :: !(Maybe AmountFraction)
+          cbFinalizationRewardCommission :: !(Maybe AmountFraction),
+          -- | Whether the account should be suspended (`Just True`) or resumed (`Just False`). `Nothing` has no effect.
```

Comment:
I guess in general, a `ConfigureBaker` payload with fields set to `Nothing` will not change the configuration?